src/app/shared/filter-bar/filter-bar.component.css

```diff
@@ -11,12 +13,29 @@
 
 .dropdown-filters {
   width: 100%;
+  display: flex;
+  flex-wrap: nowrap;
 }
 
 .label-filter-grid-tile {
   width: 100%;
+  justify-content: end;
 }
 
 .submit-button {
   margin-left: 8px;
 }
+
+.dropdown-filters-container {
+  margin-right: 8px;
+  overflow-x: auto;
+  display: flex;
+  justify-content: center;
```

Comment:
Is there a reason for justify-content and changing it when the screen is smaller? There seems to be no difference if I remove justify content in both classes in testing.